certified-connectors/tegolySIGN/apiDefinition.swagger.json

```diff
@@ -0,0 +1,164 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "tegolySIGN",
+    "description": "Automate your signatures with the tegolySIGN connector. You can transform manual processes into fully digital ones. Integrate tegolySIGN functions into your favourite applications and create fully automated signature workflows.",
```

Comment:
@JanGosda all files should free of grammatical errors and typos. please change "favourite" to "favorite"